packages/pluggableWidgets/tooltip-web/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+
+-   We fixed an issue with Tooltip widget having styling issues on a specific placement, caused due to react-popper usage.
```

Comment:
I would drop mentioning of react-popper, it doesn't say anything meaningful to our users, they probably have no idea what it is.